.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,47 @@
+---
+name: Bug report
+about: Report a defect in the software
+
+---
+
+<!--
+Thank you for your bug report.
+
+NOTE:
+
+    If you're asking about how to use OpenSSL, this isn't the right 
+    forum.  Please see our
+    [User Support resources](https://github.com/openssl/openssl/blob/master/.github/SUPPORT.md)
+
+Build issues:
+
+    If this is a build issue, please include the configuration output
+    as well as a log of all errors.  Don't forget to include the exact
+    commands you typed.
+
+    With OpenSSL before 1.1.1, the configuration output comes from the
+    configuration command.  With OpenSSL 1.1.1 and on, it's the output
+    of `perl configdata.pm --dump`
+
+Other issues: 
+
+    If it isn't a build issue, example code or commands to manifest
```

Comment:
manifest should be replaced with reproduce - it is more common english usage IMHO

Reply:
Ah, thanks!  I was wondering about that, but couldn't think of anything better at the time.  I agree with "reproduce", sounds muuuuch better.